### PR TITLE
Auto tagging

### PR DESCRIPTION
**Pull request by tomaskikutis**
SDNTB-642

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -120,7 +121,7 @@
     "sass-loader": "6.0.6",
     "shortid": "2.2.8",
     "style-loader": "0.20.2",
-    "superdesk-ui-framework": "2.2.6",
+    "superdesk-ui-framework": "github:superdesk/superdesk-ui-framework#master",
```
Comment (petrjasek) [inline package.json:124]: 2.2.11 is out now

File: scripts/extensions/auto-tagging-widget/src/auto-tagging.tsx [hunk 2]
```diff
@@ -0,0 +1,507 @@
...
+                            guid,
+                            language,
+                            headline,
+                            body_html,
+                        },
+                    },
+                }).then((res) => {
+                    const resClient = toClientFormat(res.analysis);
+
+                    this.setState({
```
Comment (petrjasek) [inline scripts/extensions/auto-tagging-widget/src/auto-tagging.tsx:103]: it should check if component is still mounted before setting the state

**Review by petrjasek — APPROVED** [review]

File: scripts/extensions/auto-tagging-widget/src/auto-tagging.tsx [hunk 3]
```diff
@@ -0,0 +1,516 @@
...
+                httpRequestJsonLocal<{analysis: IServerResponse}>({
+                    method: 'POST',
+                    path: '/ai/',
+                    payload: {
+                        service: 'imatrics',
+                        item: {
+                            guid,
+                            language,
+                            headline,
+                            body_html,
```
Comment (petrjasek) [inline scripts/extensions/auto-tagging-widget/src/auto-tagging.tsx:99]: can you add abstract here as well? or maybe just send the whole item. it's needed for SDNTB-667

Comment (tomaskikutis) [reply scripts/extensions/auto-tagging-widget/src/auto-tagging.tsx:99]: Will add after server changes in order not to break the testing instance.

Comment (petrjasek) [reply scripts/extensions/auto-tagging-widget/src/auto-tagging.tsx:99]: it's merged on the backend now

**Pull request merged**